src/DynamoCore/Utilities/Configurations.cs

```diff
@@ -22,14 +22,12 @@ public class Configurations
         public static string DynamoWikiLink = "https://github.com/DynamoDS/Dynamo/wiki";
         public static string DynamoBimForum = "http://dynamobim.org/forums/forum/dyn/";
         public static string DynamoTeamEmail = "mailto:team@dynamobim.org";
-        public static string DynamoAdvancedTutorials = "http://dynamobim.org/learn/#7570";
+        public static string DynamoAdvancedTutorials = "http://dynamobim.org/learn/#124";
```

Comment:
The start page item reads *Advanced Tutorials* but `124` leads to just the `PDF Tutorials` instead of `Advanced Tutorials` that's pointed to by `7570`. This came as [a defect fix](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-4713).